ipaserver/rpcserver.py

```diff
@@ -784,6 +785,46 @@ def marshal(self, result, error, _id=None,
         return dump.encode('utf-8')
 
 
+class jsonserver_i18n_messages(jsonserver):
+    """
+    JSON RPC server for i18n messages only.
+    """
+
+    key = '/i18n_messages'
+
+    def __init__(self, api):
+        super(jsonserver_i18n_messages, self).__init__(api)
+
+    def _on_finalize(self):
+        super(jsonserver_i18n_messages, self)._on_finalize()
+
+    def need_login(self, start_response):
+        status = '401 Unauthorized'
+        headers = []
+        response = b''
+
+        logger.debug('jsonserver_i18n_messages: %s need login', status)
+        start_response(status, headers)
+        return [response]
+
+    def __call__(self, environ, start_response):
+        '''
+        '''
+
+        logger.debug('WSGI jsonserver_i18n_messages.__call__:')
+        data = read_input(environ)
```

Comment:
Related to your Apache configuration: You should refuse all but ``GET`` requests here.

Reply:
Apache returns "403 Forbidden" as i use config access rules.

Reply:
Apache POST require rule was removed.